source/Cosmos.System2/Network/IPv4/TCP/TCPPacket.cs

```diff
@@ -409,14 +415,28 @@ internal byte[] TCP_Data
             }
         }
 
+        /// <summary>
+        /// Convert a byte to a hexadecimal char.
+        /// </summary>
+        /// <param name="b"></param>
+        /// <returns>Hex string.</returns>
+        static string ToHex(byte b)
```

Comment:
There should already be a method to do this `.ToString("X")` on a byte should return the hexadecimal representation.

Reply:
Ah, which one? This is something which should be plugged. I thought I had worked on similar plugs a short while ago.

Reply:
Need plug for: System.String  System.Number.FormatInt32(System.Int32, System.String, System.Globalization.NumberFormatInfo).
04:05:29.380991         Error:   Called from :
04:05:29.381530         Error: System.Byte::System.String ToString(System.String)

Reply:
Tracked in https://github.com/CosmosOS/Cosmos/issues/1797